### DISCUSSION TITLE
Add support Multiple Vault Passwords

### DISCUSSION THREAD

**Discussion by NikitaDef** [semaphoreui/semaphore#1805] [Ideas]
I'd like to add support the feature. I want to show my plan, which does not break compatibility.
I am open to discussion. 

## Links
[Official documentation](https://docs.ansible.com/ansible/2.9/user_guide/vault.html#vault-ids-and-multiple-vault-passwords)

## Community Issues in project 
* https://github.com/ansible-semaphore/semaphore/issues/1698

### 1. UI changes 

![image](https://github.com/ansible-semaphore/semaphore/assets/8389347/52885ae5-3375-4d80-b627-32404ff4e226)

Add and edit page Vault-IDs
![image](https://github.com/ansible-semaphore/semaphore/assets/8389347/caf70081-10b0-4934-9a65-14ef6f081459)

### 2. Database

We need to add new field into table **project__template**.

### 3. Add support into tasks

I thing I've found a place that needs a change.

https://github.com/ansible-semaphore/semaphore/blob/7c0fed0809a5ed9918fbf212ee769152ee194872/services/tasks/LocalJob.go#L240-L243

**Comment by tboerger** [comment]
Totally makes sense to support multiple vault id's.
